pygpu/collectives.pyx

```diff
@@ -228,9 +235,10 @@ cdef class GpuComm:
         """
         if dest is None:
             return pygpu_make_reduce_scattered(self, src, to_reduce_opcode(op))
-        return comm_reduce_scatter(self, src, dest, to_reduce_opcode(op))
+        comm_reduce_scatter(self, src, dest, to_reduce_opcode(op))
+        return
 
-    def broadcast(self, GpuArray array not None, int root=-1):
+    def bcast(self, GpuArray array not None, int root=-1):
```

Comment:
Ok I will return it to the full.
On Jul 8, 2016 19:29, "abergeron" <notifications@github.com> wrote:

> In pygpu/collectives.pyx
> <https://github.com/Theano/libgpuarray/pull/206#discussion_r70102185>:
>
> >
> > -    def broadcast(self, GpuArray array not None, int root=-1):
> > +    def bcast(self, GpuArray array not None, int root=-1):
>
> I'm not sure the shortened name is better.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Theano/libgpuarray/pull/206/files/dc42c5388270fadf10c9bc60588cb44e580c5825..93c5568b34db1cf5f079b91c616942e5e747c1ed#r70102185>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AD5VAZZtPlQcoDkEu2_51Ee0Le92t94wks5qTnr2gaJpZM4JAkNl>
> .
>